src/app/options.h

```diff
@@ -57,9 +59,17 @@ struct QBtCommandLineParameters
     bool portableMode;
     QString configurationName;
     QStringList torrents;
+    QString savePath;
+    TriStateBool addPaused;
+    bool skipChecking;
+    QString category;
+    bool sequential;
+    bool firstLastPiecePriority;
+    TriStateBool skipDialog;
```

Comment:
this is causing some padding bytes in the struct.
if you want to fix it (as this is not directly related to this PR), you can follow the style in `gui/advancedsettings.h`.